src/patternfly/utilities/BackgroundColor/examples/BackgroundColor.md

```diff
@@ -0,0 +1,73 @@
+---
+id: Background color
+section: utilities
+---
+
+import './BackgroundColor.css'
+
+## Examples
+
+### White
+
+```hbs
+{{#> background-color background-color--modifier="pf-u-background-color-100"}}
+  Background Color 100
+{{/background-color}}
+```
+
+### Gray
+
+```hbs
+{{#> background-color background-color--modifier="pf-u-background-color-200"}}
+  Background Color 200
+{{/background-color}}
+```
+
+### Dark 100
+
+```hbs
+{{#> background-color background-color--modifier="pf-u-background-color-dark-100"}}
+  Background Color Dark 100
+{{/background-color}}
+```
+
+### Dark 200
+
+```hbs
+{{#> background-color background-color--modifier="pf-u-background-color-dark-200"}}
+  Background Color Dark 200
+{{/background-color}}
+```
+
+### Dark 300
+
+```hbs
+{{#> background-color background-color--modifier="pf-u-background-color-dark-300"}}
+  Background Color Dark 300
+{{/background-color}}
+```
+
+### Dark 400
+
+```hbs
+{{#> background-color background-color--modifier="pf-u-background-color-dark-400"}}
+  Background Color Dark 400
+{{/background-color}}
+```
+
+## Documentation
+
+### Overview
+
+Background color utility
+
+### Usage
+
+| Class                             | Applied to | Outcome                            |
+| --------------------------------- | ---------- | ---------------------------------- |
+| `.pf-u-background-color-100`      | `*`        | Applies background-color 100.      |
```

Comment:
for these could you update the outcome to, assuming the SCSS will output the hex color

```suggestion
| `.pf-u-background-color-100`      | `*`        | Applies `--pf-global--BackgroundColor--100` (#{$pf-global--BackgroundColor--100})      |
```

Reply:
Markdown doesn't like this syntax 🙁

Reply:
Oh sorry, I was confusing mdx and SCSS. Just `--pf-global--BackgroundColor--100` is fine then!